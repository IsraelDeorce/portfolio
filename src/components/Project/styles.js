import styled from 'styled-components'

export const Project = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  max-width: 300px;
  & :first-child {
    width: 100px;
    height: 100px;
  }
`
