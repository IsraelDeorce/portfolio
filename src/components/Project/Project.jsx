import * as S from './styles'

function Project({ image, title, subtitle, description }) {
  return (
    <S.Project>
      <div>{image}</div>
      {title && <h3>{title}</h3>}
      {subtitle && <h4>{subtitle}</h4>}
      {description && <p>{description}</p>}
    </S.Project>
  )
}

export default Project
