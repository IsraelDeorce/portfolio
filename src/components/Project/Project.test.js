import Project from './Project'

describe('<Project/>', () => {
  describe('Snapshot', () => {
    it('renders without crashing using shallow', () => {
      expect(shallow(<Project />)).toMatchSnapshot()
    })
  })
})
