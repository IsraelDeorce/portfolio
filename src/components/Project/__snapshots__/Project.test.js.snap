// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Project/> Snapshot renders without crashing using mount 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  max-width: 300px;
}

.c0 :first-child {
  width: 100px;
  height: 100px;
}

<Project>
  <styled.div>
    <div
      className="c0"
    >
      <div />
    </div>
  </styled.div>
</Project>
`;

exports[`<Project/> Snapshot renders without crashing using render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  max-width: 300px;
}

.c0 :first-child {
  width: 100px;
  height: 100px;
}

<div
  class="c0"
>
  <div />
</div>
`;

exports[`<Project/> Snapshot renders without crashing using shallow 1`] = `
<styled.div>
  <div />
</styled.div>
`;
